scripts/lint.bat

```diff
@@ -8,10 +8,5 @@ cd ..
 :: Run format command in project root
 dotnet format --verbosity=d --severity=warn ChainSafe.Gaming.sln --exclude /submodules
 
-:: Navigate to the UnitySampleProject within src and run the format command
-pushd .\src\UnitySampleProject
```

Comment:
Removed this because UntiySampleProject is spitting out insane amounts of errors, and it doesn't even fix the linting in it, so it doesn't make much sense to have it here.